### PR TITLE
Fixes touch scrolling for Safari and event listener error on Chrome

### PR DESCRIPTION
**Pull request by rskarras**
Issue #227
This addresses a breaking change with newer versions of Chrome (Chrome 56 and newer) which ignores preventDefault in passive event listeners. This resulted in the following error:  

> Unable to preventDefault inside passive event listener invocation.

There is documentation on this change at
[Making touch scrolling fast by default](https://developers.google.com/web/updates/2017/01/scrolling-intervention)

This addresses issue #227 by disabling scrolling at the beginning of a drag action rather than based on styling of an element.

Issue #279
Disables touch scrolling during drag events on newer versions of Safari.

### CODE REVIEW

File: lib/DraggableCore.js [hunk 1]
```diff
@@ -229,6 +229,10 @@ export default class DraggableCore extends React.Component<DraggableCoreProps, D
   }
 
   handleDragStart: EventHandler<MouseTouchEvent> = (e) => {
+
+    // Disable touch scrolling while dragging on mobile devices.
+    document.ontouchmove = function(e){ e.preventDefault(); };
```
Comment (STRML) [inline lib/DraggableCore.js]: This makes an assumption that the developer isn't using `document.ontouchmove` for something else. Can this be done via addEventListener() instead?

Comment (STRML) [reply lib/DraggableCore.js]:
```js
const PREVENT_DEFAULT = (e) => e.preventDefault();
// ...
window.addEventListener('touchstart', PREVENT_DEFAULT, { passive: false } );
// ...
window.removeEventListener('touchstart', PREVENT_DEFAULT);
```

**Comment by rskarras** [comment]
Hi @STRML thanks for your comments and for reviewing this so quickly! You bring up a good point about document.ontouchmove being used elsewhere.

I have changed document.ontouchmove to e.target.ontouchmove and I believe that should solve this issue. 

{ return false; } is responsible for both of the following:
preventDefault - to disable scrolling following handleDragStart.
stopPropagation - to prevent the event from bubbling back up to the document level.

While I am sure it can be done with addEventListener() as suggested, it may not be necessary. 
Looking forward to hearing your thoughts on this. Thanks for your insight!

Cheers,

Sam

File: lib/DraggableCore.js [hunk 2]
```diff
@@ -229,6 +229,10 @@ export default class DraggableCore extends React.Component<DraggableCoreProps, D
   }
 
   handleDragStart: EventHandler<MouseTouchEvent> = (e) => {
+
+    // Disable touch scrolling while dragging on mobile devices.
+    e.target.ontouchmove = function(){ return false; };
```
Comment (STRML) [inline lib/DraggableCore.js:234]: This still is not removed afterwards, so it's not workable as written.

**Comment by STRML** [comment]
Fixed in #470

**Pull request closed without merging**